CONTRIBUTING.md

```diff
@@ -18,30 +25,96 @@ A post-install action is thrown. This action first install dependencies with `le
 
 At this point you are ready to contribute.
 
-## Additional installations
+## Additional tools
 
-This repository contains some projects used to display, explain components built previously.
+This repository contains some tools to help you develop on the components
 
-```
+```text
 └── storybook/
     ├── design-system : a design system web app rendering & exposing code to use components
     └── storybook : a storybook app to develop in isolated mode
 ```
 
-If you want to use these projects, you must:
+### Design System
+
+This is a special website used to display and explain components. If you are creating a new component or modifying the way a component is working, you are expected to update this website.
+
+To install it from the repository root folder (`react-toolkit`), run:
 
 ```sh
 cd storybook/design-system
 npm ci
+cd ../..
+```
+
+To launch it, you can run:
+
+```sh
+npm run design
 ```
 
+### Storybook
+
+This tool will help you display component in a page where it will be isolated (rather than integrated in a page with many components).
+
+We are using the state of the art tool: [Storybook](https://storybook.js.org/).
+
+To install it from the repository root folder (`react-toolkit`), run:
+
 ```sh
 cd storybook/storybook
 npm ci
+cd ../..
+```
+
+To launch it, you can run:
+
+```sh
+npm run storybook
+```
+
+## Keeping your forked repository up-to-date
+
+Before you start developping, it is important to always keep your forked repository **up-to-date**.
```

Comment:
Maybe you can add a link to https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo#keep-your-fork-synced

Reply:
This link is actually already shown line 9. Do you want me to repeat it here too, @xballoy ?

Reply:
It was just in case someone has already forked the repo and was only looking on how to sync it. But you can keep it like that, it was just a suggestion.

Reply:
Good idea, I added the link.